osmomath/binary_search.go

```diff
@@ -238,7 +275,7 @@ func BinarySearch(f func(Int) (Int, error),
 			return Int{}, err
 		}
 
-		compRes := errTolerance.Compare(targetOutput, curOutput)
+		compRes := Compare(errTolerance, targetOutput, curOutput)
```

Comment:
Let's not use this in the binary search code. I'm worried about state breaks, and it isn't worth the effort to sync the node for this change IMO